docs/docsite/rst/user_guide/playbooks_tags.rst

```diff
@@ -39,6 +39,8 @@ tasks, you can use the ``--skip-tags`` command-line option::
 
     ansible-playbook example.yml --skip-tags "packages"
 
+.. warning::
+    * Gathering facts about remote hosts is skipped if you run the playbook with a tag that is not defined at the playbook level.
```

Comment:
not quite, if the play has a tag defined, it affects fact gathering, if it does not, fact gathering automatically gets an 'always' tag. So you can still get facts gathered if you run a tag not on the play, as long as the play has no tags.

Reply:
@bcoca I got your point and in fact, I tried running and it is right. The playbook runs with `always` tag when no tag is used, however, the play doesn't work with other namings of the tag. Maybe the warning could be updated as below?

* When no tag is defined the playbook automatically gets an `always` tag and the facts are gathered for the remote hosts. However, gathering facts is skipped if you run the playbook with a user-defined tag defined at the playbook level.

Reply:
s/playbook/play/ you cannot tag playbooks, only plays and below, also clarify that 'when supplying tags/skip_tags, it will affect the fact gathering'.

Reply:
@bcoca, I updated the commit. Please check.